bubblewrap.c

```diff
@@ -952,7 +980,7 @@ get_newroot_path (const char *path)
   return strconcat ("/newroot/", path);
 }
 
-static void
+static void   //fix for uid maps range, instead of single will come here | but that's for later...
```

Comment:
What is this meant to mean?

This seems like it's unrelated to the rest of the PR.